### ISSUE TITLE
Combobox requires accessible name

### ISSUE THREAD

**Issue by Rawi01** [dequelabs/axe-core#1711]
Expectation: Container elements using `role="combobox"` doesn't need an `aria-label(ledby)`attribute as shown on the aria example page (https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html).

Actual: Evaluating the example page leads to violations of rule `aria-input-field-name`.

Motivation:  It seems like this worked as expected in 3.2.2 and dosen't work anymore in 3.3.0. You can check it by opening the example page and pasting 
```js
fetch("https://cdnjs.cloudflare.com/ajax/libs/axe-core/3.3.0/axe.js")
  .then(r => r.text())
  .then(eval)
  .then(() => axe.run((err, result) => console.log(result.violations)))
```
 to your browser console. The violations are gone if you change the version to `3.2.2`.

<pre><code>axe-core version: 3.3.0
Firefox
</code></pre>

**Comment by WilcoFiers** [comment]
I think that you're right. Thank you for reporting. We'll have to look into it.

**Comment by jeankaplansky** [comment]
@straker: no DQU rule help changes required for this one, correct?

**Comment by straker** [comment]
@jeankaplansky that is correct.

**Comment by chandana7393** [comment]
@straker What needs to be QA here? Can you please let me know.

**Comment by straker** [comment]
@chandana7393 The [code examples for ARIA 1.1 combobox](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html) should pass the `aria-input-field-name` rule:

```html
<label for="ex1-input" id="ex1-label">
  Choice 1 Fruit or Vegetable
</label>
<div>
  <div role="combobox" aria-expanded="false" aria-owns="ex1-listbox" aria-haspopup="listbox" id="ex1-combobox">
    <input type="text" aria-autocomplete="list" aria-controls="ex1-listbox" id="ex1-input" aria-activedescendant="">
  </div>
  <ul aria-labelledby="ex1-label" role="listbox" id="ex1-listbox"></ul>
</div>
```

**Comment by chandana7393** [comment]
Tested, working fine on Axe-coconut but in Axe and Attest etensions we are still seeing the aria-input-field-name rule.
![axe_coconut_input_label_combobox](https://user-images.githubusercontent.com/45285854/62851860-38fb7300-bd05-11e9-929e-32aee3e8251e.PNG)
![axe_input_label_combobox](https://user-images.githubusercontent.com/45285854/62851862-3ac53680-bd05-11e9-9cf7-85f153935469.PNG)
![input_label_combo](https://user-images.githubusercontent.com/45285854/62851865-3d279080-bd05-11e9-8a8d-6cfacdd806bb.PNG)

**Tested Environment:**
Attest - 2.5.1.22115v
Axe - 3.8.1.22115v
Axe-coconut - 3.8.1.22404v
Chrome - 76.0.3809.100v
Firefox - 68.0.1v
OS - Windows 10 64 bit.

**Comment by chandana7393** [comment]
Tested, working as expected.
![Attest_input_label_combobox](https://user-images.githubusercontent.com/45285854/66897556-e005cf00-f014-11e9-85e5-d2a081c19241.PNG)
![axe_input_label_combobox](https://user-images.githubusercontent.com/45285854/66897586-edbb5480-f014-11e9-8832-016053b08add.PNG)
**Tested Environment:**
Attest - 2.6.1.28822v
Axe- 4.0.0.28822v
Axe-coconut - 4.0.0.28904v
Chrome - 77.0.3865.120v
Firefox - 69.0.3v
OS  - Windows 10 64 bit.
